src/oAuth2.js

```diff
@@ -113,7 +113,8 @@ export class OAuth2 {
 
     return this.http.fetch(exchangeForTokenUrl, {
       method: 'post',
-      body: json(data),
+      headers: { 'Content-Type': 'application/json; charset=utf-8' },
+      body: JSON.stringify(data),
```

Comment:
give a read through Aurelia-fetch-client. From my reading the "json" helper should make a Blob, and then the client should add the appropriate headers. I think this is a but with Aurelia-fetch-client